ReactWindows/ReactNative.Net46/UIManager/BaseViewManager.cs

```diff
@@ -65,5 +68,42 @@ public void SetTestId(TFrameworkElement view, string testId)
         {
             AutomationProperties.SetAutomationId(view, testId ?? "");
         }
+
+        [ReactProp(ViewProps.ShadowColor, CustomType = "Color")]
+        public void SetShadowColor(TFrameworkElement view, uint? color)
+        {
+            DropShadowEffect effect = (DropShadowEffect)view.Effect ?? new DropShadowEffect();
+            effect.Color = ColorHelpers.Parse(color.Value);
+            view.Effect = effect;
+        }
+
+        [ReactProp(ViewProps.ShadowOffset)]
+        public void SetShadowOffset(TFrameworkElement view, JObject offset)
+        {
+            DropShadowEffect effect = (DropShadowEffect)view.Effect ?? new DropShadowEffect();
+            var deltaX = offset.Value<double>("width");
+            var deltaY = offset.Value<double>("height");
+            var angle = Math.Atan2(deltaY, deltaX) * (180 / Math.PI);
```

Comment:
>180 [](start = 54, length = 3)

I'm pretty sure this returns a double anyway, but it may make sense to use `180.`